docs/creating.md

```diff
@@ -127,8 +127,8 @@ Attributes:
 | `unsafe`           | `bool`                     | When `True`, allow usage of unsafe templates.<br>See the [`unsafe`][] setting for related information.                                                                                                                                                     |
 | `use_prereleases`  | `bool`                     | When `True`, `vcs_ref`/`vcs_ref_hash` may refer to a prerelease version of the template.<br>See the [`use_prereleases`][] setting for related information.                                                                                                 |
 | `user_defaults`    | `dict[str, Any]`           | Specified user defaults that may override a template's defaults during question prompts.                                                                                                                                                                   |
-| `vcs_ref`          | `str | None`               | The VCS tag/commit of the template, `None` if the template is not VCS-tracked.<br>See the [`vcs_ref`][] setting for related information.                                                                                                                   |
-| `vcs_ref_hash`     | `str | None`               | The VCS commit hash of the template, `None` if the template is not VCS-tracked.                                                                                                                                                                            |
+| `vcs_ref`          | `str                       | None`                                                                                                                                                                                                                                                      | The VCS tag/commit of the template, `None` if the template is not VCS-tracked.<br>See the [`vcs_ref`][] setting for related information. |
```

Comment:
@sisp we might have to disable `autoformat` for this PR (not sure if you can). Leaving the `\` in `str \| None` caused the `\` to actually _print_ in the assembled docs, however removing it causes `autoformat` to think that it's a column separator and add extra whitespace which then breaks the table layout 🤦‍♂️

Reply:
Mhhh, not the first time that Prettier is causing some problems with Markdown formatting. How about using HTML and entity escaping in this particular case?

```suggestion
| `vcs_ref`          | <code>str &vert; None</code>                                                                                                                                                                                                                                                            | The VCS tag/commit of the template, `None` if the template is not VCS-tracked.<br>See the [`vcs_ref`][] setting for related information. |
```

Not great, but perhaps better than disabling Prettier altogether for the table.